### PR TITLE
Sean completed react accordion

### PR DESCRIPTION
**Pull request by Sean-Hughes-1**
react-accordion

### CODE REVIEW

File: react-accordion-component/src/accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      this.setState({
+        openIndex: index
+      });
+    }
+  }
+
+  render() {
+    return (
+      <>
+        <button className="accordion" onClick={() => this.handleClick(0)}>Section 1</button>
```
Comment (Zhiwutian) [inline react-accordion-component/src/accordion.jsx:27]: while this works, you don't want to hard code the index value, you want to map over the data as an array and create the buttons. That will allow you to use the `index` parameter of a map function call

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Sean, you just want to do this without hardcoding the indexes

File: react-accordion-component/src/accordion.jsx (hunk 1 above)
Comment (Sean-Hughes-1) [reply react-accordion-component/src/accordion.jsx]: Refactored the code so that each element is dynamically rendered

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**